modules/twinklespeedy.js

```diff
@@ -1566,7 +1569,12 @@ Twinkle.speedy.callbacks = {
 				appendText += "\n\n=== " + date.getUTCMonthName() + " " + date.getUTCFullYear() + " ===";
 			}
 
-			appendText += "\n# [[:" + Morebits.pageNameNorm + "]]: ";
+			appendText += "\n# [[:" + Morebits.pageNameNorm;
+			if (params.normalizeds.indexOf("g10") === -1) {  // no article name in log for G10 taggings
```

Comment:
@Amorymeltzer @MusikAnimal TypeError: Cannot read property 'indexOf' of undefined is raised on this line when trying to CSD a file under F11 rationale (and probably with other rationales also). First reported at [TW talk page](https://en.wikipedia.org/wiki/Wikipedia_talk:Twinkle#Most_CSDs_not_being_added_to_log)

Reply:
Could've sworn I checked all the di options, but yeah, looks like I should've tested `params.normalizeds` first/as well. #460 should take care of it?  The one below this throws as well.